asklib.module

```diff
@@ -124,6 +124,43 @@ function asklib_cron() {
       $question->save();
     }
   }
+
+  // Clear contact information from 2 year old questions every day after 1 am.
+  $request_time = \Drupal::time()->getRequestTime();
+  $next_execution = \Drupal::state()->get('asklib.next_question_contact_cleanup', 0);
+
+  if($request_time > $next_execution) {
+    _asklib_clear_old_question_contact_info();
+
+    \Drupal::state()->set('asklib.next_question_contact_cleanup',
+    mktime(1, 0, 0, date("m")  , date("d")+1, date("Y")));
+  }
+  
+}
+
+function _asklib_clear_old_question_contact_info()
+{
+  $two_years_ago = mktime(1, 0, 0, date("m")  , date("d"), date("Y")-2);
```

Comment:
Do you think it would make sense to change this into a parameter that is set when the call is made so both defined periods can be found in one place?

Coding standards (same changes for the other mktime call)

```suggestion
  $two_years_ago = mktime(1, 0, 0, date("m"), date("d"), date("Y") - 2);
```